Pmods/PmodCDC1/AD7156.c

```diff
@@ -173,12 +173,14 @@ void AD7156_WriteReg(int reg, unsigned char txData)
 void AD7156_BurstWriteReg(int reg, char size, int txData)
 {
 	unsigned char txBuffer[4] = {0x00, 0x00, 0x00, 0x00};
+	int	i = 0;
 
-	txBuffer[0] = (txData & (0xFF << 8 * (size - 1))) >> (8 * (size - 1));
-	txBuffer[1] = (txData & (0xFF << 8 * (size - 2))) >> (8 * (size - 2));
-	txBuffer[2] = (txData & (0xFF << 8 * (size - 3))) >> (8 * (size - 3));
-	txBuffer[3] =  txData & (0xFF << 8 * (size - 4));
-
+	if (size < 1 || size > (sizeof(txBuffer)/sizeof(txBuffer[0])))
+		return;
+	else
+		for(i = 0; i< size; i++)
```

Comment:
nitpick:  add space between `i` & `<`  in   `for(i = 0; i< size; i++)`  --> `for(i = 0; i < size; i++)`